app/assets/stylesheets/pieces.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the pieces controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
I recommend adding a css  class to space out your piece move buttons. Something like this could work:

```
.selected-move {
  padding: 8.5px
}
```